client/my-sites/themes/controller/index.node.js

```diff
@@ -42,27 +40,14 @@ export function fetchThemeDetailsData( context, next ) {
 	}
 
 	const themeSlug = context.params.slug;
-	const theme = themeDetailsCache.get( themeSlug );
+	const theme = getThemeDetails( context.store.getState(), themeSlug );
 
-	if ( theme ) {
+	if ( Object.keys( theme ).length ) {
```

Comment:
Noting here that we discussed in Hangouts that we could try to persist cache across requests by creating the Redux store at module level (in `server/isomorphic-rendering`, much [like we did with `Map`](https://github.com/Automattic/wp-calypso/pull/3774/files#diff-3bf7293097114d293852f6d4c4d97aa7L24) up until this PR, since we don't initialize any other Redux subtrees on the server side yet.

However, I'm starting to think that this doesn't make much of a diference, and that instead, the Redux store would still be re-created on each new import of that module -- and that it's the same with the `Map` based cache now. My evidence is that after browsing two or more different theme details pages on `wpcalypso`, the `initialReduxStore` var in the page source always only holds `themeDetails` for the current theme.